index.html

```diff
@@ -1015,103 +1053,170 @@
-increasing <em>individual</em> control over [=data=]. A collective approach reduces the cost of
-control.
+## Principles for Influence and Manipulation
 
-Collecting data at large scales can have significant pro-social outcomes. Problems tend to
-emerge when entities take part in dual-use collection in which [=data=] is [=processed=]
-for collective benefit but also for [=self-dealing=] [=purposes=] that may degrade welfare.
-The [=self-dealing=] [=purposes=] will be justified as bankrolling the pro-social outcomes,
-which, absent collective oversight, cannot be considered to support claims to [=legitimacy=]
-for such [=legibility=]. It is vital for standards-setting organisations to establish not
-just purely technical devices but techno-social systems that can govern data at scale.
+<aside class="issue">
+  This section has not been written.
+</aside>
 
 <!--
-Imagine a case in which collecting browser data can be leveraged to improve both search
-services (which have a business model) and user security (which generally doesn't). This
-data could be collected through a data trust under stakeholder governance, search
-companies paying for access with strong checks and balances, and privacy guarantees, and
-the funds would be used to support enhanced security for all.
+  * Part of the threat here should include the manipulation of free and fair elections
 -->
 
-# How We Experience the World {#intrusion}
+## Principles for Harassment
 
-Privacy harms don't always come from a site learning things. For example it is
-[=intrusion|intrusive=] for a site to
+<aside class="issue">
+  This section has not been written.
+</aside>
+
+## Principles for Unwanted Information
+
+<aside class="issue">
+  This section has not been written.
+</aside>
+
+## Principles for Vulnerable People
 
-* Display messages or notifications,
-* Play sounds,
-* Occupy the full screen,
-* etc.
+Some people, or groups of people, are at greater risk from the asymmetries of power created by the
+concentration of information power ([[?Automating-Inequality]]). [=People=] who for whatever reason
+are in a more [=vulnerable=] position with respect to data [=processing=] should be more proactively
+protected by the [=user agent=] as well as by [=parties=].
+
+One issue, however, is that signalling to [=parties=] that a [=person=] may be [=vulnerable=] is
+itself a sensitive piece of information that should not be shared. One way to approach this issue is
+to rely on a [=global opt-out=] mechanism such as [[?GPC]] that is used by [=people=] whether they
+are [=vulnerable=] or not and puts a requirement on [=parties=] to minimise their [=data=]
+[=processing=].
+
+<div class="practice">
+
+<span class="practicelab" id="principle-identity-per-context">[=User agents=] should support a mode
+to specifically protect [=vulnerable=] people that defaults to sending a [=global opt-out=] signal
+such as [[?GPC]].</span>
+
+</div>
```

Comment:
I'd rather develop these new sections in dedicated PRs, rather than making this PR even bigger and slower to merge.

Reply:
Agreed. For simplicity's sake, I'll land this with the merge but then remove it to the branch.